aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -183,10 +183,20 @@ Tensor irfft(const Tensor& self, const int64_t signal_ndim, const bool normalize
               normalized, onesided);
 }
 
+template <typename Stream, typename T>
+static Stream& write_opt(Stream& SS, const optional<T>& value) {
+  if (value) {
+    SS << *value;
+  } else {
+    SS << "None";
+  }
+  return SS;
+}
 
 Tensor stft(const Tensor& self, const int64_t n_fft, const optional<int64_t> hop_lengthOpt,
```

Comment:
Let's add a comment describing the function and mentioning the LIBROSA compat we're going for

Reply:
Added a small comment.